app/templates/components/toc-section.hbs

```diff
@@ -12,8 +12,8 @@
 {{#if this.open}}
   <div>
     <ul aria-labelledby={{this.id}} class="level-3 list-unstyled">
-      {{#each @result.contents as |content|}}
-        <li>
+      {{#each @result.contents as |content index|}}
+        <li class={{if index "mt-2"}}>
```

Comment:
Not really proud of this approach. I would prefer something like `mb-2 last:mb-0`, but I could not find anything in `ember-styleguide`. Is there something available like this?

Reply:
I don't know of anything of the top of my head, unfortunately.

Reply:
I'm good with this approach though.